### ISSUE TITLE
Interstitial.setAdUnitID issue on Android

### ISSUE THREAD

**Issue by Durkaen** [NZME/react-native-ad-manager#44]
Hello,

First of all I can't thank you enough for this valuable library. I have been using it in many projects for about 2 years.

My problem is if I define once the ad id for the interstitials, I cannot replace it with a new one on Android. Let me share the code:

` 

    useEffect(() => {
    let countValue = 10;

    if (!isInterstitial) {
      Interstitial.setAdUnitID('/6499/example/interstitial');
      setInterstitial(true);
    }

    if (count >= countValue) {
      Interstitial.requestAd()
        .then(() =>
          Interstitial.showAd().catch((error) => {
            console.warn('showAd', error);
          }),
        )
        .then(() => {
          setTimeout(() => {
            Interstitial.setAdUnitID('anotherAdId');
          }, 3000);
        })
        .catch((error) => {
          setTimeout(() => {
            Interstitial.setAdUnitID('anotherAdId');
          }, 3000);
        });
    }
    }, [isInterstitial, count])
 `

In summary, I define an id when the application is first opened, and after the ad is shown, I want to define a new id. But unfortunately it still uses the old id when requesting the ads.

Everything’s fine on iOS.

**Comment by aleccolville-nzme** [comment]
thanks for using! please try again when v2.0.8 comes out
